src/make.zsh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env zsh
+
+
+_fzf_complete_make() {
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" $@ < <(grep -E '^[a-zA-Z_-]+:.*?$$' Makefile | sort | awk -F ':' '{ print $1  }')
```

Comment:
Why does it sort the result? It looks it's more obvious if they appear in the declaration order in Makefile though I'm not sure about this😉

P.S.
Nitpicking formatting issue:
`{ print $1 }`

Reply:
I have no particular opinion for sort. I'll fix it.